ntp-proto/src/packet/mod.rs

```diff
@@ -1074,30 +1159,42 @@ impl<'a> NtpPacket<'a> {
         }
     }
 
+    pub fn kiss_code(&self) -> ReferenceId {
```

Comment:
Shouldnt this return an `Option<ReferenceId>` or something like `Option<KissCode>`?

Reply:
Maybe, though I chose to just make this a private function for now.